tensorrt_llm/_torch/auto_deploy/transform/library/fused_moe.py

```
@@ -567,6 +566,176 @@ def scale_keys(self) -> List[str]:
+                "w1_weight",
+                "w2_weight",
+                "w3_weight",
+                "w1_input_scale",
+                "w2_input_scale",
+                "w3_input_scale",
+                "w1_weight_scale",
+                "w2_weight_scale",
+                "w3_weight_scale",
+            )
+        except Exception:
+            continue
```

Ruff (0.14.1)

`try`-`except`-`continue` detected, consider logging the exception

(S112)